var/spack/repos/builtin/packages/nmodl/package.py

```diff
@@ -19,6 +19,7 @@ class Nmodl(CMakePackage):
     version('0.2', tag='0.2', submodules=True)
 
     variant("legacy-unit", default=True, description="Enable legacy units")
+    variant("pybind11", default=False, description="Enable python bindings")
```

Comment:
We should rename this to `python-bindings`. (because pybind11 is still used for embedded python and solver stuff when python-bindings are disabled)

Reply:
Why not just `+python` as is done normally?

Reply:
ok, +python